scipy-1.0/paper.tex

```diff
@@ -975,73 +968,54 @@ \subsection*{Polynomial interpolators}
 
 \subsection*{Test and benchmark suite}
 
-\subsubsection*{Benchmark suite}
-
-The airspeed velocity (asv\cite{asvref}) library enables benchmarking Python packages
-over their lifetimes, and the performance of the SciPy code base was monitored
-with asv starting in February of 2015 (PR \#4501). In addition to ensuring that
-unit tests are passing (see Test suite section below), confirming 
-that performance generally
-remains constant or improves over the commit hash history of the project allows
-us to objectively measure that our code base is improving, to empower
-scientific applications.
-
-Consider the asv benchmark results shown in Figure~\ref{fig:asvbench}, spanning
-roughly nine years of project history. These demonstrate the gradual
-performance improvements in a nearest-neighbor search through
-\texttt{scipy.spatial.cKDTree.query()}, and can be run using:
-
-\texttt{python run.py run -e -s 800 --bench "\textbackslash
-btime\_query\textbackslash b" "02de46a546..b3ddb2c"},
-
-where \texttt{run.py} is a light wrapper script around \texttt{asv}
-that is packaged with SciPy.
-
-\begin{figure}[H]
-\centering
-\includegraphics[width=\textwidth]{static/asv_time_query_ckdtree}
-\caption{Airspeed velocity benchmarks for scipy.spatial.cKDTree.query()
-over a roughly nine year commit history time frame. The results are based on
-Python 2.7 performance on the master branch of the project using NumPy 1.8.2
-and Cython versions 0.27.3, 0.21.1, and 0.18 (for improved backward
-compatibility). Only the L2 (Euclidean) norm is shown here, and to improve
-backward compatibility and sampling of the benchmarks there was no application
-of toroidal topology to the KDTree (boxsize argument was ignored).}
-\label{fig:asvbench}
-\end{figure}
-
-Any pull request can be compared against the \texttt{master} branch with the
-command \texttt{asv continuous master new-feature}. This will provide a
-benchmark against the master branch and the branch a new feature is implemented
-in. More features are available in the \texttt{asv} documentation\cite{asvdocs}, including 
-arguments to select which benchmarks to run.
-
 \subsubsection*{Test suite}
+\label{sec:testsuite}
 
 The SciPy test suite is orchestrated by a continuous integration matrix that
 includes posix and Windows (32/64-bit) platforms managed by Travis CI and
-appveyor, respectively. Our tests cover Python versions 2.7, 3.4, 3.5, 3.6, and
```

Comment:
What is new?

Reply:
If you're not sure, then maybe the issue already opened -- #97 regarding the stats section -- is the place to discuss this first. I think @chrisb83 has opened the issue.

Reply:
All of it -- CI is relatively new

Reply:
See the timeline figure you made

Reply:
That's why I asked. The timeline shows Travis CI starting in 2013. The section is about
"key technical improvements made in the last three years". Since we started Travis CI more than three years before the release of SciPy 1.0 in late 2017, I thought it was worth checking about the others.